src/foam/nanos/controller/AppStyles.js

```diff
@@ -272,11 +272,53 @@ foam.CLASS({
     font-weight: bold;
     font-size: 2.8rem;
   }
-  @media only screen and (min-width: 321px) {
+
+  /* mobile */
+  @media only screen and (min-width: 321px) and (max-width: 767px) {
```

Comment:
Don't need `max-width`, here and elsewhere

Reply:
changed to select max-width instead since min-width will also override styles for desktop sizes